Provide section around mass contributions

weierophinney opened the pull request:
To prevent issues with unwanted mass patches and/or propagating incorrect patches, we need  contributors to get permission from and coordinate with the TSC prior to submission.

weierophinney committed:
Provide section around mass contributions

To prevent issues with unwanted mass patches and/or propagating incorrect patches, we need  contributors to get permission from and coordinate with the TSC prior to submission.

Signed-Off-By: Matthew Weier O'Phinney <matthew@weierophinney.net>

michalbundyra commented on CONTRIBUTING.md:
```diff
@@ -264,5 +264,26 @@ Date:   Wed Apr 01 14:46:40 2020 -0500
+We do not want to waste either maintainer or contributor time, so coordinating such changes, while it may seem like an extra step, can save everybody time and effort in the long run.
+
+If you are unsure whether to start pushing patches to all repositories, please open an agenda item for the next TSC meeting.
+If you need assistance doing so, ask in the #contributors channel.
```
would be nice if we can have a link to `#contributors` slack channel (I think it's possible to do somehow with `slack://`, but not sure what it's gonna be).

ref: https://api.slack.com/reference/deep-linking#client

weierophinney replied:
Problem is that only works for people using an official client. If you're using via the web, or in a wrapper like Rambox, those links won't work.

boesing commented on CONTRIBUTING.md:
```diff
@@ -264,5 +264,26 @@ Date:   Wed Apr 01 14:46:40 2020 -0500
+
+We do not want to waste either maintainer or contributor time, so coordinating such changes, while it may seem like an extra step, can save everybody time and effort in the long run.
+
+If you are unsure whether to start pushing patches to all repositories, please open an agenda item for the next TSC meeting.
```
Hm, I'd be okay with an RFC Issue in the TSC as well. If it turns out that we need a broader discussion, an additional agenda item could still be created but given the fact that the TSC meeting is once a month, that might be a bit too long just to get feedback on an idea? 🤷🏼‍♂️

weierophinney replied:
Good point; updating to require an RFC first.

weierophinney committed:
Reference RFC, not agenda

Updates to reference creating an RFC, and not an agenda item, when proposing mass changes.

Signed-Off-By: Matthew Weier O'Phinney <matthew@weierophinney.net>

weierophinney commented:
Changes also pushed to laminas-api-tools and mezzio orgs.
